### PR TITLE
ElectraForMultipleChoice

### PR DESCRIPTION
**Pull request by patil-suraj**
This PR add `ElectraForMultipleChoice`. One of the missing models in this [project](https://github.com/huggingface/transformers/projects/17).

Since, for multiple choice pooled outputs are needed, added `ElectraPooler` class.

@sgugger , @LysandreJik

### CODE REVIEW

File: src/transformers/modeling_electra.py [hunk 1]
```diff
@@ -858,3 +858,133 @@ def forward(
             outputs = (total_loss,) + outputs
 
         return outputs  # (loss), start_logits, end_logits, (hidden_states), (attentions)
+
+
+class ElectraPooler(nn.Module):
```
Comment (sgugger) [inline src/transformers/modeling_electra.py:863]: Instead of introducing a new class, I'd rather use the existing `SequenceSummary` which can do the pool + dropout (see the [`XLNetForMultipleChoice`](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_xlnet.py#L1369) for an example. It will require to add a few things to the config to make it work (see `XLNetConfig` for an example).

**Review by sgugger — CHANGES_REQUESTED** [review]
Thanks for your PR! I think there is a way to make it more consistent with the other models for multiple choice and avoid introducing a new class, would you mind looking into it?

File: src/transformers/modeling_electra.py (hunk 1 above)
Comment (patil-suraj) [reply src/transformers/modeling_electra.py:863]: >I think there is a way to make it more consistent with the other models for multiple choice and avoid introducing a new class, would you mind looking into it?

Sure, I wasn't aware of `SequenceSummary`.  I'll try to use it instead of `Pooler`

Comment (stefan-it) [reply src/transformers/modeling_electra.py:863]: But couldn't you just use the `ElectraClassificationHead` for that pooling 🤔:

https://github.com/huggingface/transformers/blob/86578bb04c9b34f9d8e35cd4fad42a85910dd9e9/src/transformers/modeling_electra.py#L346-L362

Comment (sgugger) [reply src/transformers/modeling_electra.py:863]: Sadly no, because it adds a linear layer to `num_labels`

**Review by sgugger — APPROVED** [review]
LGTM, @LysandreJik it's good for your review.

**Review by LysandreJik — APPROVED** [review]
LGTM, thanks @patil-suraj

**Comment by patil-suraj** [comment]
hi @LysandreJik  could you tell me why these tests are failing now ? Thanks.

**Comment by sgugger** [comment]
Looks like @LysandreJik deleted a function by mistake during the merge (the `create_and_check_electra_for_multiple_choice`), could you add it back?

**Comment by LysandreJik** [comment]
My bad, sorry about that. Thanks for the fix!
